contracts/common/governance/Governable.sol

```diff
@@ -10,10 +10,14 @@ contract Governable {
     }
 
     modifier onlyGovernance() {
+        _assertGovernance();
```

Comment:
what's the difference?

Reply:
Modifier copies it's content to the function. Instead of copying revert messages everywhere (which occupies precious 24kb) let's copy function call. Tiny more gas and huge bytecode savings

Reply:
yeah cool.